docs/ops_guide/repush.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Repush
+parent: Operator Guides
+permalink: docs/ops_guide/repush
+---
+
+# Repush
+
+In Venice, there are many things which happen under the hood as part of Full Pushes. Besides serving to refresh data,
+Full Pushes also serve to make certain configs take effect, and also more generally keep the system tidy and healthy.
+
+Many Venice users choose to periodically do Full Pushes for the sake of refreshing their data, in which case the
+auxiliary goals of Full Pushes end up being fulfilled organically.
+
+In other cases, users do Full Pushes either infrequently, or not at all. In these cases, the operator may want to 
+manually trigger a Repush, without involving the user.
+
+## Use Cases
+The various motivations for operator-driven Repushes are described in more details below.
+
+### Reconfiguration
+In Venice, some store configs can be changed live, while some others take effect when the next store-version is pushed.
+Configs which can be changed with a Repush (or a regular Full Push) include:
+
+1. Partition-related settings
+   1. Partition count
+   2. Amplification factor
+   3. Custom partitioner implementation
+2. Replication settings
+3. Whether a store is hybrid
```

Comment:
Add enable chunking

Reply:
We discussed this in another comment thread, and Nisarg told me we had disabled support for changing chunking as part of a repush.

Reply:
Seems to be correct based on looking at the VPJ code here: https://github.com/linkedin/venice/blob/master/clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java#L2132